isa/kernel.org/instruction-set.rst

```diff
@@ -318,49 +446,221 @@ the atomics features, while keeping a lower ``-mcpu`` version, you can use
+0x97    any   dst %= imm                                          `Arithmetic instructions`_
+0x9c    0x00  dst = (uint32_t)(dst % src)                         `Arithmetic instructions`_
+0x9f    0x00  dst %= src                                          `Arithmetic instructions`_
+0xa4    any   dst = (uint32_t)(dst ^ imm)                         `Arithmetic instructions`_
+0xa5    any   if dst < imm goto +offset                           `Jump instructions`_
+0xa6    any   if (uint32_t)dst < imm goto +offset                 `Jump instructions`_
+0xa7    any   dst ^= imm                                          `Arithmetic instructions`_
+0xac    0x00  dst = (uint32_t)(dst ^ src)                         `Arithmetic instructions`_
+0xad    0x00  if dst < src goto +offset                           `Jump instructions`_
+0xae    0x00  if (uint32_t)dst < (uint32_t)src goto +offset       `Jump instructions`_
+0xaf    0x00  dst ^= src                                          `Arithmetic instructions`_
+0xb4    any   dst = (uint32_t) imm                                `Arithmetic instructions`_
+0xb5    any   if dst <= imm goto +offset                          `Jump instructions`_
+0xa6    any   if (uint32_t)dst <= imm goto +offset                `Jump instructions`_
+0xb7    any   dst = imm                                           `Arithmetic instructions`_
+0xbc    0x00  dst = (uint32_t) src                                `Arithmetic instructions`_
+0xbd    0x00  if dst <= src goto +offset                          `Jump instructions`_
+0xbe    0x00  if (uint32_t)dst <= (uint32_t)src goto +offset      `Jump instructions`_
+0xbf    0x00  dst = src                                           `Arithmetic instructions`_
+0xc3    0x00  lock *(uint32_t *)(dst + offset) += src             `Atomic operations`_
+0xc3    0x01  lock *(uint32_t *)(dst + offset) += src             `Atomic operations`_
+              lock src = *(uint32_t *)(dst + offset)
+0xc3    0x40  *(uint32_t *)(dst + offset) |= src                  `Atomic operations`_
+0xc3    0x41  *(uint32_t *)(dst + offset) |= src                  `Atomic operations`_
+              lock src = *(uint32_t *)(dst + offset)
+0xc3    0x50  *(uint32_t *)(dst + offset) &= src                  `Atomic operations`_
+0xc3    0x51  *(uint32_t *)(dst + offset) &= src                  `Atomic operations`_
+              lock src = *(uint32_t *)(dst + offset)
+0xc3    0xa0  *(uint32_t *)(dst + offset) ^= src                  `Atomic operations`_
+0xc3    0xa1  *(uint32_t *)(dst + offset) ^= src                  `Atomic operations`_
+              lock src = *(uint32_t *)(dst + offset)
+0xc3    0xe1  swap(src, *(uint32_t *)(dst + offset))              `Atomic operations`_
+0xc3    0xf1  lock temp = *(uint32_t *)(dst + offset)             `Atomic operations`_
+              if *(uint32_t)(dst + offset) == R0
+                 *(uint32_t)(dst + offset) = src
```

Comment:
This indent renders as a quote in RST. Same for `0xdb`.

Reply:
Yeah, I'm still fiddling with the formatting of this one.  Suggestions welcome.

Reply:
This should now be fixed.

Reply:
Looks good!

In the list of RST nitpicks, there are also tables or item lists that are apparently considered as quotes, because of their indent.

Reply:
Can you point me to a specific example to look at?

Reply:
The unordered list at https://github.com/dthaler/ebpf-docs/blob/new/isa/kernel.org/instruction-set.rst#12registers-and-calling-convention; or compare the two tables at https://github.com/dthaler/ebpf-docs/blob/new/isa/kernel.org/instruction-set.rst#14arithmetic-and-jump-instructions (before 1.4.1): The first one is in a block quote (looks dimmed, with a vertical bar on the left), the second is not. They both have indent, but the first one is indented with respect to regular paragraphs (making it a quote block) whereas the second is aligned with (has the same level of indent than) the description for the `source` field.

Let me know if you would like me to create a commit to address it somehow.